File: src/Page/MyStudentsPage.java
package Page;

import SystemMessage.FunctionType;
import SystemMessage.SystemMessage;

public class MyStudentsPage extends Page {
	
	public MyStudentsPage(String content) {
		super(content);
		setType(PageType.MY_STUDENTS_PAGE);
		setName("All Students");
	}
	
	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, null);
	}
}

File: python_development/Page/MyStudentsPage.py
from python_development.Page.Page import Page
from python_development.Page import PageType
from python_development.System import FunctionType
from python_development.System import SystemMessage


class MyStudentsPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType(PageType.MY_STUDENTS_PAGE)
        self.setName("All Students Page")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, None)
